### DISCUSSION TITLE
[Projects Beta] Search for issues with no BETA Project

### DISCUSSION THREAD

**Discussion by gflarity** [community/community#8911] [Projects and Issues]
I want to be able to search for issues that aren't in a beta project yet. With regular projects you just go no:project, but that doesn't work with beta projects :(

**Comment by Totktonada** [comment]
Asked the same here: https://github.com/github/feedback/discussions/5856

**Comment by eelco** [comment]
Similar to @Totktonada’s issue, this bug is making our triaging flow harder. Hopefully it can be fixed soon? 🙏
